### ISSUE TITLE
Loaded FluxDistributions not displayed

### ISSUE THREAD

**Issue by matthiaskoenig** [matthiaskoenig/cy2fluxviz#1]
After loading flux distributions, these are not displayed.
Reproducible with Java 8 on Windows and Ubuntu 64bit.

see also discussion
https://groups.google.com/forum/#!topic/cytoscape-discuss/4yh_MK7Dkv8

```
cy2sbml[INFO]: cy2sbml-v1.4.0
cy2sbml[CONFIG]: Initialization finished
CyFluxViz[INFO] -> cyfluxviz-v0.95
CyFluxViz[INFO]: GUI components initialized.
CyFluxViz[INFO]: initNodeAttributeComboBox()
CyFluxViz[INFO]: initNodeAttributeComboBox()
cy2sbml[CONFIG]: createVisualStyle( cy2sbml )
cy2sbml[INFO]: Preload Miriam for compartments
cy2sbml[INFO]: Preload Miriam for species
cy2sbml[INFO]: Preload Miriam for reactions
CyFluxViz[INFO]: initNodeAttributeComboBox()
Message adding FluxDistributions: ''
Exception in thread "AWT-EventQueue-0" java.lang.UnsupportedOperationException
    at java.util.AbstractList.remove(AbstractList.java:161)
    at java.util.AbstractList$Itr.remove(AbstractList.java:374)
    at java.util.AbstractList.removeRange(AbstractList.java:571)
    at java.util.AbstractList.clear(AbstractList.java:234)
    at browser.AttributeBrowser.getSelectedAttributes(AttributeBrowser.java:278)
    at cyfluxviz.FluxDisCyAttributes.selectCyFluxVizTableAttributes(Unknown Source)
    at cyfluxviz.gui.CyFluxVizPanel.updateFluxDistributionTable(Unknown Source)
    at cyfluxviz.FluxDisCollection.addFluxDistributions(Unknown Source)
    at cyfluxviz.gui.CyFluxVizPanel$5.actionPerformed(Unknown Source)
    at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022)
    at javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348)
    at javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402)
    at javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259)
    at javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252)
    at java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:289)
    at java.awt.Component.processMouseEvent(Component.java:6535)
    at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
    at java.awt.Component.processEvent(Component.java:6300)
    at java.awt.Container.processEvent(Container.java:2236)
    at java.awt.Component.dispatchEventImpl(Component.java:4891)
    at java.awt.Container.dispatchEventImpl(Container.java:2294)
    at java.awt.Component.dispatchEvent(Component.java:4713)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
    at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
    at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
    at java.awt.Container.dispatchEventImpl(Container.java:2280)
    at java.awt.Window.dispatchEventImpl(Window.java:2750)
    at java.awt.Component.dispatchEvent(Component.java:4713)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
    at java.awt.EventQueue.access$500(EventQueue.java:97)
    at java.awt.EventQueue$3.run(EventQueue.java:709)
    at java.awt.EventQueue$3.run(EventQueue.java:703)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
    at java.awt.EventQueue$4.run(EventQueue.java:731)
    at java.awt.EventQueue$4.run(EventQueue.java:729)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by matthiaskoenig** [comment]
Problem also existing in cy2sbml. Probably due to some change in the cytoscape.AttributeBrowser between Cytoscape v2.8.2 -> v2.8.3.

**Comment by matthiaskoenig** [comment]
Fixed with latest cy2sbml release v1.4.1
